### PR TITLE
Amazon SageMaker

### PR DESCRIPTION
**Pull request by tescal2**
This is a pull request to add reference to Amazon SageMaker from the READ.ME page as well as the updated wiki page that uses an updated walk through using the Image Classification sample notebook within Amazon SageMaker Studio.

### CODE REVIEW

**Comment by adrianboguszewski** [comment]
Thanks for this PR. Your changes should be also included in notebooks/README.md file. You refer to the AWS wiki, which doesn't exist. Are you going to provide the content for that page?

**Comment by tescal2** [comment]
> 



> Thanks for this PR. Your changes should be also included in notebooks/README.md file. You refer to the AWS wiki, which doesn't exist. Are you going to provide the content for that page?

@adrianboguszewski Great callout. I went ahead and committed the update to the notebooks/READ.ME as well as part of the PR. Also, the wiki page is authored and can be found here: https://github.com/tescal2/openvino_notebooks/wiki/Amazon-SageMaker

We will need to then remove an outdated version that already exists found here: https://github.com/openvinotoolkit/openvino_notebooks/wiki/Sagemaker

File: README.md
Comment (adrianboguszewski) [inline README.md:27]:
```suggestion
	* [Amazon SageMaker](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Amazon-SageMaker)
```

File: README.md
Comment (adrianboguszewski) [inline README.md:48]:
```suggestion
| [Windows](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Windows) | [Ubuntu](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Ubuntu) | [macOS](https://github.com/openvinotoolkit/openvino_notebooks/wiki/macOS) | [Red Hat](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Red-Hat-and-CentOS) | [CentOS](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Red-Hat-and-CentOS) | [Azure ML](https://github.com/openvinotoolkit/openvino_notebooks/wiki/AzureML) | [Docker](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Docker) | [Amazon SageMaker](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Amazon-SageMaker)|
```

File: notebooks/README.md
Comment (adrianboguszewski) [inline notebooks/README.md:27]:
```suggestion
	* [Amazon SageMaker](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Amazon-SageMaker)
```

File: notebooks/README.md
Comment (adrianboguszewski) [inline notebooks/README.md:48]:
```suggestion
| [Windows](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Windows) | [Ubuntu](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Ubuntu) | [macOS](https://github.com/openvinotoolkit/openvino_notebooks/wiki/macOS) | [Red Hat](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Red-Hat-and-CentOS) | [CentOS](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Red-Hat-and-CentOS) | [Azure ML](https://github.com/openvinotoolkit/openvino_notebooks/wiki/AzureML) | [Docker](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Docker) | [Amazon SageMaker](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Amazon-SageMaker)|
```

**Review by adrianboguszewski — CHANGES_REQUESTED** [review]
Is it possible to get the raw version of https://github.com/tescal2/openvino_notebooks/wiki/Amazon-SageMaker? It would be easier to copy it to our wiki

**Comment by tescal2** [comment]
> @adrianboguszewski Great callout. I went ahead and

Does this work? https://raw.githubusercontent.com/wiki/tescal2/openvino_notebooks/Amazon-SageMaker.md

**Comment by adrianboguszewski** [comment]
Yes. Thank you. Please let us test these instructions before we merge this PR.

**Comment by ryanloney** [comment]
@tescal2 @adrianboguszewski 
Unfortunately, at this time, openvino-dev is built following manylinux_2_27 standard. This means pip install openvino-dev does not work on Amazon Linux. See below: 
image

We have an internal request to try building manylinux_2_17 (aka manylinux2014) python wheels using gcc10 for openvino-dev. Until we do this, you will need to build from source code. These are the instructions to do it: https://github.com/openvinotoolkit/openvino/wiki/Building-OpenVINO-on-CentOS-7-Guide

Until we can pip install openvino-dev on Amazon Linux, I don't think it makes sense to expose this guide. 

Anyone else have input? I hope we can get the install working soon!

-Ryan

**Comment by ryanloney** [comment]
@tescal2 sorry for the confusion. We see the process we tested was outdated. Updating the wiki to use your new instructions. Then we can merge this

**Review by ryanloney — APPROVED** [review]
I updated the wiki page. @ravi9 is testing on his SageMaker instance, but these instructions were tested externally yesterday. I'm good to merge this as soon as one more approver reviews. 

@adrianboguszewski  @raymondlo84 can you take a look. Here is updated guide: https://github.com/openvinotoolkit/openvino_notebooks/wiki/SageMaker

**Comment by ryanloney** [comment]
Ravi made updates to the guide. Let's get this merged tomorrow, if possible.

**Review by adrianboguszewski — APPROVED** [review]

**Pull request merged**
